cmd/snap/cmd_snap_op.go

```diff
@@ -680,6 +680,11 @@ func (x *cmdTry) Execute([]string) error {
 	}
 
 	changeID, err := cli.Try(path, opts)
+	if e, ok := err.(*client.Error); ok && e.Kind == client.ErrorKindNoSnapDir {
+		return fmt.Errorf(i18n.G(`%q does not contain an unpacked snap.
+
+Try "snapcraft prime" in your project directory, then "snap try prime"`), path)
```

Comment:
`then "snap try prime"` => `then "snap try" again.`

We seem to handle the prime case fine, per logic right above. Bonus points for the "try again" pun. ;D